### ISSUE TITLE
Panic when launching on Linux x86_64

### ISSUE THREAD

**Issue by noodler9** [xbapps/xbvr#297]
I'm trying xbvr for the first time. Both versions I have tried, 0.3.0-beta.27 and 0.3.0-beta.28 for Linux x86_64, segfault when launching, with the same error:

`./xbvr
[0000]  INFO Migration did run successfully
[0000]  INFO ffprobe not installed, downloading now...
[0000]  INFO Starting router
[0000]  INFO Added realm: default
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x80 pc=0x9a0aa6]

goroutine 234 [running]:
github.com/gammazero/nexus/v3/client.(*Client).Connected(...)
        /go/pkg/mod/github.com/gammazero/nexus/v3@v3.0.0/client/client.go:126
github.com/gammazero/nexus/v3/client.(*Client).Publish(0x0, 0x141c9a4, 0xb, 0x0, 0x0, 0x0, 0x0, 0xc0015760f0, 0x203000, 0x12)
        /go/pkg/mod/github.com/gammazero/nexus/v3@v3.0.0/client/client.go:292 +0x26
github.com/xbapps/xbvr/pkg/common.(*WampHook).Fire(0xc001496fc0, 0xc001580150, 0x4, 0xc00157e088)
        /go/appsrc/pkg/common/logging.go:36 +0x2c1
github.com/sirupsen/logrus.LevelHooks.Fire(0xc00037af30, 0x4, 0xc001580150, 0x4e7e0b08aa320, 0xc00083ee16)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.6.0/hooks.go:28 +0x91
github.com/sirupsen/logrus.(*Entry).fireHooks(0xc001580150)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.6.0/entry.go:266 +0x83
github.com/sirupsen/logrus.Entry.log(0x2f79c60, 0xc0015760c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.6.0/entry.go:244 +0x110
github.com/sirupsen/logrus.(*Entry).Log(0xc0015800e0, 0x4, 0xc001544e88, 0x1, 0x1)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.6.0/entry.go:287 +0xed
github.com/sirupsen/logrus.(*Entry).Logln(0xc0015800e0, 0xc000000004, 0xc001576090, 0x3, 0x3)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.6.0/entry.go:378 +0xd3
github.com/sirupsen/logrus.(*Entry).Infoln(...)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.6.0/entry.go:391
github.com/sirupsen/logrus.(*Entry).Println(...)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.6.0/entry.go:395
github.com/sirupsen/logrus.(*Logger).Println(0x2f79c60, 0xc001576090, 0x3, 0x3)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.6.0/logger.go:259 +0x6a
github.com/gammazero/nexus/v3/router.(*router).Close.func1()
        /go/pkg/mod/github.com/gammazero/nexus/v3@v3.0.0/router/router.go:276 +0xb6
github.com/gammazero/nexus/v3/router.(*router).run(0xc00151aff0)
        /go/pkg/mod/github.com/gammazero/nexus/v3@v3.0.0/router/router.go:357 +0x49
created by github.com/gammazero/nexus/v3/router.NewRouter
        /go/pkg/mod/github.com/gammazero/nexus/v3@v3.0.0/router/router.go:90 +0x21f`

This is on an up to date Debian 10 box.

**Comment by atani12** [comment]
Beta-28 works fine for me, but I'm using a different distro.

Try deleting the ~/.config/xbvr folder first and then start beta-28 again to see if the same errors occur.

You can always go for the docker option, which is the option I normally use, and that should be independent of your system.

**Comment by noodler9** [comment]
> Try deleting the ~/.config/xbvr folder first and then start beta-28 again to
> see if the same errors occur.

Thanks for the suggestion, I did try this, but it made no difference 
unfortunately.

Doing more tests, this happens for me every release down 'til at 
least 0.1.4! I tried to download the go project to debug myself, but 
I'm on Go 1.11 here, so some of the scrapers failed to build. I'll 
try to get past that and debug further in due course - if anyone has 
any hints, though, that'd be great - I'm not familiar with Nexus, so 
it may take me some time to figure out...
